migrations/versions/2019_04_18_add_topic_short_title.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
```

Comment:
Be nice to replace the `empty message` with a message.

Reply:
👍